app/views/simulator/edit.html.erb

```diff
@@ -212,7 +212,7 @@
         </div>
       </div>
       <div id="combinationalAnalysis" style="display:none;" title="BooleanLogicTable"></div>
-      <div id="insertSubcircuitDialog" style="display:none;" title="Insert SubCircuit"></div>
+      <div id="insertSubcircuitDialog" style="display:none;overflow-x:hidden;overflow-y:auto" title="Insert SubCircuit"></div>
```

Comment:
css please

Reply:
Their is no change in css @tachyons  the subcircuit dialog   is generated using js

Reply:
The internal css needs to be transferred to respective external css file @yash15601.

Reply:
I have not made any changes  in css it was already

Reply:
Please do it anyways. Thanks.